build/docker-extension/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  portainer:
+    image: ${DESKTOP_PLUGIN_IMAGE}
+    command: ['--admin-password', '$$$$2a$$$$10$$$$HX4qSZhtQcvKUNmAsPXuPe9POkM7gdaFPcSnRjokgb8EkMI.1gkSa']
```

Comment:
do we have a document to tell what password correspond to that hash?

Reply:
No that I am aware of, good point

Reply:
FYI its `Passw0rd;`

Reply:
passed this on to support to be documented

Reply:
they don't need to login, we skip login for the extension